index.html

```diff
@@ -0,0 +1,45 @@
+<script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script> 
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Homenaje a Fulano de Tal</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+  </head>
+  
+  <body>
+    <div id="main">
+      
+      <div class="header">
+        <h2 id="title">Fulano de Tal</h2>
+      </div>
+      <div class="buttons" z-index: 2>
+        <div class="menubutton" id="button1">Inicio</div>
+        <div class="menubutton" id="button2">Retrato</div>
+        <div class="menubutton" id="button3">Biografía</div>
+        <div class="menubutton" id="button4">Enlaces</div>
+      </div>
+      
+      <div class="content">
+        <div id="img-div"><img id="image" src="https://i.ytimg.com/vi/cEJKfqU5efA/maxresdefault.jpg" alt="Fulano de Tal en una entrevista televisiva.">
+          <div id="img-caption"><p>Arriba: <strong>Fulano de Tal</strong> en una entrevista televisiva.</p></div>
+        </div>
+        
+        <div id="tribute-info"><p>Aquí va el texto de la página</p></div>
+        <div><p>Aquí va <a id="tribute-link" href="https://es.wikipedia.org/wiki/Sin_nombre" target="_blank"> el enlace de la página</a></p></div>
+      </div>
+    </div>
+  </body>
+</html>
+
+
+
+<!-- 
```

Comment:
Delete unnecessary comment. This is like Mari Kondo, if it's clutter and doesn't bring joy, it should go. Be merciless.